include/regs.h

```diff
@@ -58,15 +58,26 @@
 #define R_FB_FILTER_ENV_COLOR(c)                 SREG(85 + (c))
 #define R_ENABLE_FB_FILTER                       SREG(88)
 #define R_PAUSE_MENU_MODE                        SREG(94)
+#define R_CAM_XZ_UPDATE_RATE                     OREG(2)
+#define R_CAM_Y_OFFSET_UPDATE_RATE               OREG(3)
+#define R_CAM_FOV_UPDATE_RATE                    OREG(4)
 #define R_CAM_MAX_PITCH                          OREG(5)
+#define R_CAM_R_UPDATE_RATE_INV                  OREG(6)
 #define R_CAM_DEFAULT_PITCH_UPDATE_RATE_INV      OREG(7)
+#define R_CAM_SLOPE_Y_ADJ_ADJ_AMOUNT             OREG(9)
```

Comment:
The double `_ADJ_ADJ_` looks awkward, although I see where it comes from `FUNC_NAME`_`ARG_NAME`. Could `AdjAmt` arg be renamed to something else perhaps?

Reply:
tbh `R_CAM_SLOPE_Y_ADJ_AMOUNT` would be fine, I'll change it